### DISCUSSION TITLE
[ 2021 / 04 / 24 ] Community Call Discussions

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1389] [Community Call]
Hey feel free to discuss the topics mentioned in the community call here

**Comment by eddiejaoude** [comment]
Previous month https://github.com/EddieHubCommunity/support/discussions/1176

- Feel free to reply to people's threads with thoughts, ideas, support, questions, something you learnt etc.
- If you reply in the voice call, please also leave a response on their thread so it is not lost
- Please remember our [Code of Conduct](https://github.com/EddieHubCommunity/support/blob/main/CODE_OF_CONDUCT.md).

**Comment by eddiejaoude** [comment]
Give yourself a shout out here 👇

**Reply by muthuannamalai12** [reply]
Hello Everyone,
I am Muthu Annamalai, currently pursuing BE in Computer Science and Engineering from PEC Chennai. Being a newbie, I am still learning and therefore I try to use every opportunity to know more about technology. Though I am new, I can vouch for the fact that I am a fast learner with a penchant to both learn and unlearn. I am currently into learning web development
Also, you can find me here:
Linkedin Profile: https://www.linkedin.com/in/muthu-annamalai/
Git Hub: https://github.com/muthuannamalai12
Twitter: https://twitter.com/muthuannamalai_

**Reply by muthuannamalai12** [reply]
I stammered a lot please bear with me as I am new to public speaking

**Reply by eddiejaoude** [reply]
You were great 👍

**Reply by kaiwalyakoparkar** [reply]
you were great @muthuannamalai12

**Reply by illusiveCode** [reply]
Hi, I am Raza! 

Here to learn by collaborating, and I glad to be part of EddieHub :)
Would love to connect with everyone who loves sharing knowledge and learning.

Reach out here : -

Twitter: https://twitter.com/illusiveCode
LinkedIn: https://www.linkedin.com/in/frontend-developer-muhammad-raza/

**Reply by Rahma-AlWadhahi** [reply]
Hello everyone,
I’m Rahma. I’m trying my best to get involved in open source 😊 and have a healthy looking profile with green squares 🟩
I’m working as a software developer in Oman.

**Reply by vinzvinci** [reply]
Hey folks. I'm Vincent Villafuerte, I do like collaboration and contributing to the open-source. You can visit my GitHub profile to know me well, there's a lot of information about me there. Feel free to ping me if you have any questions.
Looking forward to having a good conversation with you all on the EddieHub discord server. Thank you.

Link: <a href="https://github.com/vinzvinci">https://github.com/vinzvinci</a>

**Comment by eddiejaoude** [comment]
"Chill out time" Kunal

- don't burn out
- take breaks

**Reply by illusiveCode** [reply]
- Reward yourself

**Reply by kunal-kushwaha** [reply]
Set realistic goals

**Reply by kunal-kushwaha** [reply]
Don’t compare yourself with others

learn from other’s experiences 

join communities to learn

**Reply by eddiejaoude** [reply]
Oh yes, join communities! 💪

**Comment by eddiejaoude** [comment]
"Public speaking" Karuna

- people want you to succeed
- practice practice

**Reply by illusiveCode** [reply]
- The main tip is in the title, 'Speak in public' as much as possible
- Write a script then read from it

**Reply by DevCassie** [reply]
I just read about this topic and I remembered a video on public speaking from Simon Sinek. Public Speaking isn't always needing to read from a script according to Simon Sinek. Which I agree with. Making mistakes is part of being a human being. Practice is indeed good for public speaking. But public speaking is also about capturing the public.

The video from Simon Sinek that I remembered:
https://www.youtube.com/watch?v=962eYqe--Yc

**Comment by muthuannamalai12** [comment]
I am new to the content creation world can I have any ideas to create content as I always do not get ideas to post

**Reply by DevCassie** [reply]
I'm aspiring to make some content soon. What helped me was to just create content that you enjoy. It's always better to do something that you enjoy.

**Reply by muthuannamalai12** [reply]
Thank you for your suggestion

**Reply by eddiejaoude** [reply]
Document what you are doing and learning, content creation is hard, but content documentation is much easier because you are doing it anyway 🤓 

I only put out about 40% of the content I create 😱

**Reply by eddiejaoude** [reply]
Learning in public 👍  (by @kunal-kushwaha )

**Reply by eddiejaoude** [reply]
It is marathon not a sprint 🤓

**Comment by eddiejaoude** [comment]
How to remember to say "hey folks" by @Akshu-on-github 

- practice
- it is difficult at the beginning
- friend reminding us

**Comment by ashwinexe** [comment]
Hi Everyone!
I am Ashwin!  2nd year Engineering Student at SNIST, Hyderabad. I like Competitive Programming and Web Development. But I'm open to learn more feel free to talk to me. Right now I'm focusing on web development and wanting to learn open source!

**Reply by eddiejaoude** [reply]
please can you move to this thread https://github.com/EddieHubCommunity/support/discussions/1389#discussioncomment-653920

**Comment by Rahma-AlWadhahi** [comment]
I have a suggestion that I got just now.
What if we get top three topics that the community will vote on to talk about in the next community call rather than staying without questions? 
Maybe keeping them as spare topics or making the call to be theme related? 🤔

**Reply by eddiejaoude** [reply]
good idea, keeping it more focused might be good 👍 - I am happy to try it 🤓

**Reply by illusiveCode** [reply]
This would be a great idea. Like have a poll for topics to talk about maybe...
